plugins/CMakeLists.txt

```diff
@@ -60,14 +60,13 @@ elseif("${CMAKE_SYSTEM_NAME}" MATCHES "OpenBSD")
 	add_subdirectory(sndio)
 endif()
 
-option(BUILD_BROWSER "Build browser plugin" OFF)
+option(BUILD_BROWSER "Build browser plugin" ON)
```

Comment:
Is this supposed to be here? Enabling the browser is more involving than changing this boolean because it involves setting a path for CEF, which is hard to do automatically. Maybe split the changes to defaults as another pull request so we can discuss it separatedly?

Reply:
It's meant to "force" Linux packagers to see that there is submodules to set up. Because many packagers doesn't seem to know about them. I may need to make a PR for obs-browser to make it emit also a fatal error if CEF is not found.

Reply:
Hmmmmm... usually, distros take a tarball generated, for example, by `make distcheck` or `ninja dist`, which should in theory have submodules. I'm not sure what's the release process for OBS, or if Jim rolls out manual tarballs. If packagers are expected to use git tags, than this makes sense. But regardless of that, I guess it's making sure packagers either explicitly disable it, or make sure it actually really works, is a good idea.

Reply:
No submodules in github source code tarball

Reply:
[obs-browser PR](https://github.com/obsproject/obs-browser/pull/285)